config/config.exs

```diff
@@ -14,6 +14,11 @@ config :dotcom, :location_service, LocationService
 
 config :dotcom, :mbta_api_module, MBTA.Api
 
+config :dotcom, :mbta_go_app,
+  android_store_base_url: "https://play.google.com/store/apps/details?id=com.mbta.tid.mbta_app",
+  ios_store_base_url: "https://apps.apple.com/app/apple-store/id6472726821?",
+  default_project_page: "/goapp"
+
```

Comment:
suggestion: I think you can leave this out the config and put these in `DotcomWeb.AppStoreController` directly, since they aren't needed anywhere else. I'm not convinced it's worth the trouble mocking these values in tests either -- the iOS test could look for `"apps.apple.com"` in the link, for example.